src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.css

```diff
@@ -9,8 +9,19 @@
   margin-left: 20px;
 }
 
+.tokenAmountContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 20px;
+  gap: 20px;
+}
+
 .tokenAmountUsd {
-  justify-self: right;
+  width: 100%;
+  font-size: 11px !important;
```

Comment:
I don't like the use of important. Have you tried other ways of raising specificity like: 
`.tokenAmountUSD > span`

Reply:
I've removed the !important from both tokenAmountUsd, since I made size optional and we're no longer passing small size appearance as props to EthUsd component, the size is correct at 11px without need of further specyfiying.

Does that work?

Reply:
yes, very good